src/server/services/readMamacMeters.js

```diff
@@ -7,7 +7,7 @@ const readCsv = require('./readCsv');
 const parseString = require('xml2js').parseString;
 const Meter = require('../models/Meter');
 const util = require('util');
-const _ = require('lodash');
+const zipObject = require('lodash/zipObject');
```

Comment:
I don't think zipObject is used so it can be removed.

Also, stopDB is not used if you want to remove that too.

Reply:
Sorry for the delay, I think I have resolved this issue. I forgot to remove the _ in _.zipObject, so it wasn't being called properly. I also removed the stopDB import.